src/transformers/models/convbert/tokenization_convbert.py

```diff
@@ -45,14 +50,466 @@
 }
 
 
-class ConvBertTokenizer(BertTokenizer):
+# Copied from transformers.models.bert.tokenization_bert.load_vocab
+def load_vocab(vocab_file):
+    """Loads a vocabulary file into a dictionary."""
+    vocab = collections.OrderedDict()
+    with open(vocab_file, "r", encoding="utf-8") as reader:
+        tokens = reader.readlines()
+    for index, token in enumerate(tokens):
+        token = token.rstrip("\n")
+        vocab[token] = index
+    return vocab
+
+
+# Copied from transformers.models.bert.tokenization_bert.whitespace_tokenize
+def whitespace_tokenize(text):
+    """Runs basic whitespace cleaning and splitting on a piece of text."""
+    text = text.strip()
+    if not text:
+        return []
+    tokens = text.split()
+    return tokens
+
+
+# Copied from transformers.models.bert.tokenization_bert.BertTokenizer with ConvBertTokenizer->BertTokenizer
+class ConvBertTokenizer(PreTrainedTokenizer):
     r"""
-    Construct a ConvBERT tokenizer. [`ConvBertTokenizer`] is identical to [`BertTokenizer`] and runs end-to-end
-    tokenization: punctuation splitting and wordpiece. Refer to superclass [`BertTokenizer`] for usage examples and
-    documentation concerning parameters.
+    Construct a BERT tokenizer. Based on WordPiece.
```

Comment:
As said in the comment above, we need a BERT->ConvBERT replacement for the docstrings.